Packages/KeyAppKit/Sources/BankTransfer/Service/StrigaMockBankTransferService.swift

```diff
@@ -3,6 +3,8 @@ import Foundation
 
 public final class StrigaMockBankTransferService: BankTransferService {
 
+    private var _userData: UserData = .init(countryCode: "fr", userId: nil, mobileVerified: false)
```

Comment:
don't you think we need to start add logic to real service? without accessing to server yet